MTGToolbox.Core/MTGToolbox.Core.csproj

```diff
@@ -1,7 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I also moved everything except the web project to NET Standard because there's no need for the Core SDK. Optional but it ensures your stuff is super compatible should you ever want to reuse any of this. And if you ever need to move it back to Core, it's as simple as editing this line (for the most part).